api/v2/index.html

```diff
@@ -1,18 +1,18 @@
 <html>
 <body>
 
-<p>You probably have landed here because you are using the old URL associated to the NGSIv2 Specification :). The spec is now <a href="stable">here</a>.</p>
+<p>You probably arrived here looking for the NGSIv2 Specification document :)</p>
 
-<!--p>You probably have landed here because you are using the old URL associated to the NGSIv2 Specification :).</p>
+<p>Instead of that, you should use the <a href="TBD">Orion API Specification</a>, which is based in the original NGSIv2 specification
```

Comment:
```suggestion
<p>Instead of that, you should use the <a href="https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md">Orion API Specification</a>, which is based in the original NGSIv2 specification
```